From: drtr@ast.cam.ac.uk (David Robinson)
Subject: Set AUTH_TYPE variable for CGI scripts
Affects: http_protocol.c
ChangeLog: Fix get_basic_auth_pw() to set the auth_type of the request.

*** http_protocol.c.orig3	Wed Dec 13 14:08:49 1995
--- http_protocol.c	Thu Dec 14 10:40:47 1995
***************
*** 379,384 ****
--- 379,385 ----
  
      t = uudecode (r->pool, auth_line);
      r->connection->user = getword (r->pool, &t, ':');
+     r->connection->auth_type = "Basic";
  
      *pw = t;
  
